### PR TITLE
Task Order 24 cleaned changes

### PR DESCRIPTION
**Pull request by rlevy12**
This PR represents the cleaned-up and combined changes to Student with respect to Task Order 24. This is done to eliminate the noise caused by auto formatting in earlier versions of these changed files, which complicated comparing and merging.

### CODE REVIEW

File: student/src/main/java/tds/student/web/controls/dummy/ResourcesLink.java [hunk 1]
```diff
@@ -85,32 +126,49 @@ public void encodeAll (FacesContext context) throws IOException {
...
-        scriptLink.setSource ( UrlHelper.buildUrl (scriptLinkUrl, fileInput.getPath ()));
-        scriptLink.encodeAll (context);
+				String path = fileInput.tryGetPathRelativeTo(rootVirtualPath, new PathFormatter());
+
+				if (path == null)
+					continue;
+
+				UIComponentBase control = createControl(path);
+				/*
+				 * ScriptLink scriptLink = new ScriptLink();
```
Comment (hbgwhite) [inline student/src/main/java/tds/student/web/controls/dummy/ResourcesLink.java]: Please do not check in commented code.

File: student/src/main/webapp/Projects/SBAC/Help/help_streamlined.html [hunk 2]
```diff
@@ -0,0 +1,725 @@
...
+    
+</ul>
+
+
+
+  <div class="inner">
+  <span class="horizontal3"></span>
+   
+
+<!--SECTION 1-->
```
Comment (hbgwhite) [inline student/src/main/webapp/Projects/SBAC/Help/help_streamlined.html:42]: Please do not check in commented code

File: student/src/main/webapp/Shared/CSS/Universal/shell.css [hunk 3]
```diff
@@ -27,8 +27,8 @@ body {
 	overflow:hidden;
```
Comment (hbgwhite) [inline student/src/main/webapp/Shared/CSS/Universal/shell.css:27]: It appears that this file was formatted outside of the areas where actual changes were made. These formatting changes make it very difficult to tell what actually changed. Please re-submit this file without the formatting.

File: student/src/main/webapp/resources/sections/TTSCheck.xhtml [hunk 4]
```diff
@@ -1,122 +1,186 @@
...
+						<span lang="es-mx" style="display: none"
+							i18n-content="TTSCheck.Label.HearMessageSpanish"></span>
 					</div>
 					<p i18n-content="TTSCheck.Label.HeardYesOrNoSpanish"></p>
 				</div>
-				<span class="clear"></span> <span class="actions"> <tds:Button id="btnSoundSpanishNo" styleClass="negative" key="TTSCheck.Label.HeardNoSpanish" /> <tds:Button id="btnSoundSpanishYes"
-						styleClass="positive" key="TTSCheck.Label.HeardYesSpanish" /> <tds:Button id="skipSpanish" styleClass="positive" key="TTSCheck.Label.SkipSpanish" />
-				</span> <span class="clear"></span>
+				<span class="clear"></span>
+				<tds:NextStepWrapper id="sectionCheckSoundSpanishNextSteps"
```
Comment (hbgwhite) [inline student/src/main/webapp/resources/sections/TTSCheck.xhtml:66]: Are there changes here or is this just formatting? Please submit a clearer diff.

**Review by hbgwhite — CHANGES_REQUESTED** [review]
There are some files in this pull request that appear to have been formatted independent of any changes. It's difficult to review these files. Please re-submit the changes without formatting.

There are some files that look as if code has been commented out in the commits and then checked in. If the code is no longer in use, please remove it.

File: student/src/main/webapp/resources/sections/TTSCheck.xhtml (hunk 4 above)
Comment (vkesharia) [reply student/src/main/webapp/resources/sections/TTSCheck.xhtml:66]: This are actual code changes

File: student/src/main/webapp/Shared/CSS/Universal/shell.css (hunk 3 above)
Comment (vkesharia) [reply student/src/main/webapp/Shared/CSS/Universal/shell.css:27]: Below commit is the actual code change
https://github.com/SmarterApp/TDS_Student/commit/7578c162c4a582e1d1d71b92e6763d57fcbfd9d0?diff=split

File: student/src/main/java/tds/student/web/controls/dummy/ResourcesLink.java (hunk 1 above)
Comment (vkesharia) [reply student/src/main/java/tds/student/web/controls/dummy/ResourcesLink.java]: Will will remove commented out code and commit

File: student/src/main/webapp/Projects/SBAC/Help/help_streamlined.html (hunk 2 above)
Comment (vkesharia) [reply student/src/main/webapp/Projects/SBAC/Help/help_streamlined.html:42]: Will will remove commented out code and commit

File: student/src/main/webapp/Projects/SBAC/Help/help.html [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+          <td class="toolleft"><strong>Help button <img src="images/HelpButton.png" alt="question mark icon" style="vertical-align:middle"/></strong></td>
+          <td class="toolRight">To view the on-screen <strong><em>Test Instructions and Help</em></strong> window, select the question mark <img src="images/HelpButton.png" alt="question mark icon" style="vertical-align:middle"/> button in the upper-right corner.</td>
+        </tr>
+        <tr>
+          <td class="toolleft"><strong>Zoom buttons</strong></td>
+          <td class="toolRight">Use the [<strong>Zoom In</strong>] button to make the text and images bigger. Use the [<strong>Zoom Out</strong>] button to make the text and images smaller. There are four zoom levels.</td>
+        </tr>
+          <!--<tr>
+          <td class="toolleft"><strong>Line Reader</strong></td>
+          <td class="toolRight">Use the [<strong>Line Reader</strong>] button to highlight a line of text on the page. When you click this button, the first line of text on the page will be highlighted blue. Click another line of text on the page to highlight it.
```
Comment (hbgwhite) [inline student/src/main/webapp/Projects/SBAC/Help/help.html]: Please remove commented sections

File: student/src/main/webapp/Projects/SBAC/Help/help_esn.html [hunk 6]
```diff
@@ -0,0 +1,248 @@
...
+        </tr>
+          <tr>
+          <td class="toolleft"><strong>Botón de Ayuda <img src="images/HelpButton.png" alt="signo de interrogación" style="vertical-align:middle"/></strong></td>
+          <td class="toolRight">El botón de ayuda <img src="images/HelpButton.png" alt="signo de interrogación" style="vertical-align:middle"/> en la parte superior derecha accede a esta página de instrucciones de pruebas y a la página de ayuda.</td>
+        </tr>
+         <tr>
+          <td class="toolleft"><strong>Botones de Zoom</strong></td>
+          <td class="toolRight">Utilice el botón [<strong>Acercar</strong>] para ampliar el texto y las imágenes. Utilice el botón [<strong>Alejar</strong>] para hacerlas más pequeñas. Puede acercar y alejar el texto y las imágenes hasta cuatro veces.</td>
+        </tr>
+          <!--<tr>
```
Comment (hbgwhite) [inline student/src/main/webapp/Projects/SBAC/Help/help_esn.html]: Please remove commented sections

File: student/src/main/webapp/Projects/SBAC/Help/help.html (hunk 5 above)
Comment (vkesharia) [reply student/src/main/webapp/Projects/SBAC/Help/help.html]: https://github.com/SmarterApp/TDS_Student/commit/534f9518227418dacff23eea853aff4175c39d14

File: student/src/main/webapp/Projects/SBAC/Help/help_esn.html (hunk 6 above)
Comment (vkesharia) [reply student/src/main/webapp/Projects/SBAC/Help/help_esn.html]: https://github.com/SmarterApp/TDS_Student/commit/534f9518227418dacff23eea853aff4175c39d14

**Comment by jtreuting** [comment]
Completed as part of https://github.com/SmarterApp/TDS_Student/pull/34

**Pull request closed without merging**